src/app/modules/recipe-form/components/recipe-view/recipe-view.component.ts

```diff
@@ -0,0 +1,74 @@
+import {Component, Input, OnInit, Output, EventEmitter} from '@angular/core';
+import {Emitted, Recipe} from "../../models/recipe-create.model";
+import {RecipeFormService} from "../../services/recipe-form.service";
+import {ActivatedRoute, Router} from "@angular/router";
+
+@Component({
+  selector: 'recipe-view',
+  templateUrl: './recipe-view.component.html',
+  styleUrls: ['./recipe-view.component.scss']
+})
+export class RecipeViewComponent implements OnInit {
+
+  @Input() form_name: string[] = ['',''];
```

Comment:
split to two @inputs 
```suggestion
  @Input() formName: string = '';
  @Input() actionButtonName: string = '';
```